### PR TITLE
feat: about footer migrated to Mantine

### PR DESCRIPTION
**Pull request by yasmin-e**
Closes: #4894 

### Description:
1. modals with/without update
Screenshot 2023-03-27 at 17 20 51
Screenshot 2023-03-27 at 17 21 19
2. footer on pages
Screenshot 2023-03-27 at 17 18 33
Screenshot 2023-03-27 at 17 15 22
3. footer on sidebar (SQL runner)
Screenshot 2023-03-27 at 17 15 44
Screenshot 2023-03-27 at 17 25 55

### CODE REVIEW

**Comment by PriPatel** [comment]
To be clear, the footers on content pages and SQL runner sidebars only appear in the demo right?

**Review by IrakliJani — APPROVED** [review]
just one comment about container size: otherwise looks great. https://lightdash.slack.com/archives/C02GQKJK84Q/p1679930874363609

File: packages/frontend/package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
         "@mantine/hooks": "^6.0.4",
         "@sentry/react": "^7.37.2",
         "@sentry/tracing": "^7.37.2",
-        "@tabler/icons-react": "^2.1.1",
+        "@tabler/icons-react": "^2.11.0",
```
Comment (IrakliJani) [inline packages/frontend/package.json:22]: these libraries are moving fast 🏃 🏃 🏃

**Pull request merged**